src/selenium.ts

```diff
@@ -14,9 +14,10 @@ import * as selenium from 'selenium-standalone';
 import * as which from 'which';
 import * as child_process from 'child_process';
 import * as wct from 'wct';
-import * as promisify from 'promisify-node';
+// import promisify from 'util';
+const { promisify } = require("util");
 
-const SELENIUM_OVERRIDES: any = require(`${process.env.PWD}/package.json`)['selenium-overrides'] || any = require('../package.json')['selenium-overrides'];
+const SELENIUM_OVERRIDES: any = require(`${process.env.PWD}/package.json`)['selenium-overrides'] || require('../package.json')['selenium-overrides'];
```

Comment:
May I suggest to get selenium-overrides from wct.conf instead of package.json.

Reply:
That could be a perfectly valid place to get it from. My changes in this file weren't for usability, but simply because it's invalid typescript, thus wouldn't transpile.